execution_workflows/IsoSCM/modules/isoscm_compare.nf

```diff
@@ -0,0 +1,28 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process ISOSCM_COMPARE {
```

Comment:
Can you add some description to this step as in the assemble step? 
Is this where two samples/conditions are needed and we the same?